tests/test_lib_tokens_daypassword.py

```diff
@@ -414,97 +398,83 @@ def test_18_challenges(self):
         self.assertTrue(resp, resp)
 
         # test if challenge is valid
-        chal.is_valid()
+        self.assertTrue(chal.is_valid())
+        chal.delete()
 
-    @mock.patch('time.time', mock.MagicMock(return_value=1686902767))
+#    @mock.patch('time.time', mock.MagicMock(return_value=1417549521))
```

Comment:
maybe we should remove this

Reply:
Done in [107a4f3](https://github.com/privacyidea/privacyidea/pull/3735/commits/107a4f35ffefc12471b6d7c7798c6c8345f5cbf0)
(force-pushed)